check_stability.py

```diff
@@ -622,7 +622,7 @@ def get_parser():
                         action="store",
                         # Travis docs say do not depend on USER env variable.
                         # This is a workaround to get what should be the same value
-                        default=os.environ.get("TRAVIS_REPO_SLUG").split('/')[0],
+                        default=(os.environ.get("TRAVIS_REPO_SLUG") or "").split('/')[0],
```

Comment:
This doesn't really make sense, does it? If you don't pass anything here you will get a runtime error. So we should either check that or make the default `w3c` i.e. 

`default=os.environ.get("TRAVIS_REPO_SLUG", "w3c").split("/")[0]`